mining-paper.tex

```diff
@@ -266,21 +262,21 @@ \subsection{Restake mining compensation}
     \label{fig:total-compensation}
 \end{figure}
 
-\subsection{Take mining compensation and spindown}
+\subsection{Take compensation and spindown}
 
-When the node spins down, the miner doesn't extend the time for end of staking $t_1$,
+When the node spins down, the staker doesn't extend the time for end of staking $t_1$,
 and the compensation is constantly decreasing as the time left to unlock becomes smaller and smaller,
 effectively decreasing $\kappa$ gradually towards $0.5$.
-That's the default behavior: to avoid that, the miner should set $t_1$ large enough, or increase $t_1$ periodically.
+That's the default behavior: to avoid that, the staker should set $t_1$ large enough, or increase $t_1$ periodically.
```

Comment:
This is no longer the default...The default is that wind down is disabled.

Reply:
```suggestion
The default setting for all stakes is for $t_1$ to be automatically prolonged, which maintains the maximum reward rate ($\kappa$ = 1). Stakers wishing to withdraw must proactively initiate a wind down on some or all of their stake.
```